.travis.yml

```diff
@@ -19,6 +19,9 @@ script:
   - docker-compose up --build --force-recreate -d
   - sleep 20
   - wget --no-check-certificate https://eid.local
+  - docker-compose exec -T api sh -c 'python manage.py creatersakey; (exit $?)'
+  - docker-compose exec -T api sh -c 'python manage.py registerclient "testclient" "123456" "" "public" "id_token token" "http://testclient"; (exit $?)' 
```

Comment:
some parts of the test code use `CLIENT_ID` 123456, other parts use the env variable. Please confirm that the tests are working even if `.env` file sets `CLIENT_ID` to a value that different from 123456.